video/out/vo_gpu_next.c

```diff
@@ -1696,31 +1658,25 @@ static int preinit(struct vo *vo)
     ra_hwdec_ctx_init(&p->hwdec_ctx, vo->hwdec_devs, gl_opts->hwdec_interop, false);
     pthread_mutex_init(&p->dr_lock, NULL);
 
-#if PL_API_VER >= 320
-    p->cache = pl_cache_create(pl_cache_params(
+    p->shader_cache = pl_cache_create(pl_cache_params(
         .log = p->pllog,
-        .max_object_size =  1 << 20, //  1 MB
-        .max_total_size  = 10 << 20, // 10 MB
+        .max_object_size =  1 << 20, //  1 MiB
+        .max_total_size  = 10 << 20, // 10 MiB
```

Comment:
This should be 50 MiB and without per object size limit. https://code.videolan.org/videolan/libplacebo/-/commit/ea91056682d1117a2b70b8a2c17773180aa3ac2c

gamut mapping luts are bigger, than 1 MiB and overall 10 MiB is not enough. 0/50 MiB is reasonable limit.

Reply:
Sure.